product_docs/docs/pem/8/changing_default_port.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Changing the default port"
+---
+
+After installation, configure the PEM server. On Windows, configuration occurs automatically. On Linux, configure the server by running a script.
```

Comment:
I think we could do a better job describing the intent of this topic. There are other topics that describe how to configure the server. This topic describes how to change the default port. What is significant about the Windows config being automatic and Linux being script-based? Are this instructions Linux-specific? It seems so. Should we instead say something like: After configuring the PEM server on Linux, you can change the default port.